src/services/paras/ParasService.ts

```diff
@@ -1,41 +1,67 @@
-import { Option } from '@polkadot/types/codec';
-import { BlockHash, FundInfo, ParaId } from '@polkadot/types/interfaces';
+import { Option, Vec } from '@polkadot/types/codec';
+import { AbstractInt } from '@polkadot/types/codec/AbstractInt';
+import {
+	AccountId,
+	BalanceOf,
+	BlockHash,
+	FundInfo,
+	ParaGenesisArgs,
+	ParaId,
+	ParaLifecycle,
+	WinningData,
+} from '@polkadot/types/interfaces';
+import { ITuple } from '@polkadot/types/types';
+import BN from 'bn.js';
 import { InternalServerError } from 'http-errors';
 
 import {
-	ICrowdloansInfoResponse,
-	ICrowdloansResponse,
-	IEntries,
+	AuctionPhase,
+	IAuctionsCurrent,
+	ICrowdloans,
+	ICrowdloansInfo,
+	IFund,
+	ILeaseInfo,
+	ILeasesCurrent,
+	IParas,
+	LeaseFormatted,
+	ParaType,
 } from '../../types/responses';
+import { IOption, isSome } from '../../types/util';
 import { AbstractService } from '../AbstractService';
 
 export class ParasService extends AbstractService {
 	/**
-	 * @param hash
-	 * @param paraId
-	 * @returns
-	 * {
-	 *   "at": {},
-	 *   "paraId": number,
-	 *   "fundInfo": {...}
-	 * }
+	 * @param hash `BlockHash` to make call at
```

Comment:
```suggestion
         *  `/:paraId/crowdloan-info` Service function to retrieve crowdloan information for a specific paraId.
	 * @param hash `BlockHash` to make call at
```

Maybe something like this instead of the `@returns`?